.circleci/generate.go

```diff
@@ -0,0 +1,146 @@
+// +build generate
+
+// This program prints the CircleCI configuration for the "pkg" repository. Standard way to run it is to run
+// "go run generate.go".
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"text/template"
+)
+
+const (
+	header = `checkout-path: &checkout-path
+  checkout-path: /go/src/github.com/palantir/pkg
```

Comment:
Any value in proving modules are doing their thing by not using a GOPATH?

Reply:
In Go 1.13 and above, if there's a `go.mod` present, module mode will always be used (even in GOPATH). We still need to keep the checkout path here in this commit since we have the non-module packages (which will be converted in next pass).